### PR TITLE
EVEREST-269 Renamed ObjectStorage to BackupStorage

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

File: config/crd/bases/everest.percona.com_databaseclusterrestores.yaml [hunk 1]
```diff
@@ -132,7 +132,7 @@ spec:
                   sslSecretName:
                     type: string
                   storage_type:
-                    description: BackupStorageType represents backup storage type.
+                    description: BackupStorageType is a type of object storage.
```
Comment (oksana-grishchenko) [inline config/crd/bases/everest.percona.com_databaseclusterrestores.yaml:135]: there shouldn't be "object storage" I believe

File: bundle/manifests/everest.percona.com_databaseclusterrestores.yaml [hunk 2]
```diff
@@ -131,7 +131,7 @@ spec:
                   sslSecretName:
                     type: string
                   storage_type:
-                    description: BackupStorageType represents backup storage type.
+                    description: BackupStorageType is a type of object storage.
```
Comment (oksana-grishchenko) [inline bundle/manifests/everest.percona.com_databaseclusterrestores.yaml:134]: there shouldn't be "object storage" I believe

**Review by oksana-grishchenko — COMMENTED** [review]
Thanks for implementing this! 
There are still some mentions of "object storage" in comments and error messages, please change it too.

**Review by recharte — APPROVED** [review]
Good job, don't forget to follow up the merge of this one with a PR to update the backend.

**Comment by oksana-grishchenko** [comment]
Some leftovers still can be found 
Screenshot 2023-08-14 at 15 03 39

**Comment by gen1us2k** [comment]
@oksana-grishchenko Fixed everything

**Pull request merged**